src/pybind/mgr/dashboard/frontend/src/app/shared/services/login.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
```

Comment:
A few things: `login.service` is rather misleading, as this has little to do with the `auth/login` part of the code, so that could confuse people.

On the other hand, this service actually takes lots of information pieces from other services and adds little value (no processing or transformation, just the info grouped), so my suggestion would be to put all this stuff into the about box component itself, and remove the login.service. I don't think this service could be easily reusable by someone else.